storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="persistent-storage-csi-smb-cifs"]
+= CIFS/SMB CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-smb-cifs
+
+toc::[]
+
+{product-title} is capable of provisioning persistent volumes (PVs) with a Container Storage Interface (CSI) driver for Common Internet File System (CIFS) dialect/Server Message Block (SMB) protocol.
+
+:FeatureName: CIFS/SMB CSI Driver Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+After installing the CIFS/SMB CSI Driver Operator, {product-title} installs corresponding pods for the operator and driver in the `openshift-cluster-csi-drivers` namespace by default. This allows the CIFS/SMB CSI Driver to create CSI-provisioned persistent volumes (PVs) that mount to CIFS/SMB shares.
+
+* The _CIFS/SMB CSI Driver Operator_, after being installed, does not create a storage class by default to use to create persistent volume claims (PVCs). However, xref:../../storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc#persistent-storage-csi-smb-cifs-create-sc_persistent-storage-csi-smb-cifs[you can manually create the CIFS/SMB `StorageClass`]. The CIFS/SMB CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand.
+This eliminates the need for cluster administrators to pre-provision storage.
+
+* The _CIFS/SMB CSI driver_ enables you to create and mount CIFS/SMB PVs.
```

Comment:
Same question as above about italicization, but I'm guessing it's to introduce and differentiate terms